### ISSUE TITLE
Select set autoComplete="off" not works

### ISSUE THREAD

**Issue by wegorich** [mantinedev/mantine#841]
### What package has an issue

@mantine/core

### Describe the bug

I have no possibility to turn off the native autocomplete in select. 

I'm doing this way, but seems control not act with autocomplete prop.

```
  <Select
    zIndex={10000}
    autoComplete="off"
    searchable
    data={agroCulturesOptions}
  />
```

at the compiled code I see the:

```
<input autocomplete="nope" type="text" id="listId" aria-autocomplete="list" data-mantine-stop-propagation="false" name="listId" aria-invalid="false" class="mantine-Select-defaultVariant mantine-Select-input mantine-1tge4fo" value="">
```

### In which browser did the problem occur

chrome

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Possible fix

Push forward autocomplete prop

**Comment by rtivital** [comment]
Usually, browsers ignore `autocomplete="off"` on inputs and show native autocomplete anyway, so setting it not `nope` solves the issue.

**Comment by wegorich** [comment]
<img width="249" alt="Снимок экрана 2022-02-16 в 13 40 43" src="https://user-images.githubusercontent.com/1983361/154222599-ef0c1461-3822-4bd8-9d2e-e5f9fb238b3d.png">

But it makes select unusable, the issues is. I even cant control the autocomplete prop in component. To avoid the case. Autcomplete works well on other components from this kit.

**Comment by rtivital** [comment]
What can you propose to solve this issue?

**Comment by wegorich** [comment]
put the value from <Select autoComplete=[VALUE]> to the <input autoComplete=[VALUE]/> without any magic beside

**Comment by rtivital** [comment]
Okay, I'll add the option to override `autoComplete` prop on Select in next patch

**Comment by wegorich** [comment]
I found that the issue is happen only when the `searchable` flag is on. 

Thanks ;)

**Comment by rtivital** [comment]
Released in 3.6.8, you can now override autocomplete attribute like in Autocomplete component
